internal/exec/describe_stacks.go

```diff
@@ -307,15 +308,22 @@ func ExecuteDescribeStacks(
 							if err != nil {
 								return nil, err
 							}
-						} else {
+						} else if atmosConfig.Stacks.NamePattern != "" {
```

Comment:
@aknysh please check this part. We need to ensure that both name pattern and name templates work.